pytorch_lightning/accelerators/dp_accelerator.py

```diff
@@ -172,3 +174,9 @@ def reinit_scheduler_properties(self, optimizers: list, schedulers: list):
                 scheduler.__class__.__mro__[idx].__init__(scheduler, optimizer)
                 if state is not None:
                     scheduler.load_state_dict(state)
+
+    def reference_model(self, model: Union[LightningDataParallel, LightningModule]) -> LightningModule:
```

Comment:
shall it be rather a property or static?
since `self.trainer.model` is defined, this makes me think why it class method if `self` is not needed

Reply:
I wasn't able to make this a property, as the model isn't accessible from `self.trainer.model` in setting up ddp_train.